### DISCUSSION TITLE
Just wondering, what is Github Actions and how do we use it?

### DISCUSSION THREAD

**Discussion by penguinmoose** [theforumhelpers/theforumhelpers.github.io#323] [General]
I saw that we use Github Actions (correct me if we don't). What is it and what do we use it for?

**Comment by Accio1** [comment]
GitHub Actions is a feature that GitHub offers to automate parts of the development process. Currently, we use GitHub Actions to scan our code for security risks. In the past, we used it to auto-label issues and pull requests, but it stopped working so we disabled it.

**Reply by penguinmoose** [reply]
Ok, thanks!
